From b12df30e8ddb680200cc4de7439eda476115ab15 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Mon, 7 May 2018 11:32:11 +0300
Subject: [PATCH] arm: dts: armada-38x.dtsi: use control0 in thermal sensor

The patch enlarges second resource in the Armada 38x thermal sensor dt node.
This triggers the priv->control0 code in drivers/thermal/armada_thermal.c.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/boot/dts/armada-38x.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/armada-38x.dtsi b/arch/arm/boot/dts/armada-38x.dtsi
index f88acb1..e33d18a 100644
--- a/arch/arm/boot/dts/armada-38x.dtsi
+++ b/arch/arm/boot/dts/armada-38x.dtsi
@@ -595,7 +595,7 @@
 
 			thermal: thermal@e8078 {
 				compatible = "marvell,armada380-thermal";
-				reg = <0xe4078 0x4>, <0xe4074 0x4>;
+				reg = <0xe4078 0x4>, <0xe4070 0x8>;
 				status = "okay";
 			};
 
-- 
1.7.9.5

